examples/quick-start-typescript/index.ts

```diff
@@ -0,0 +1,66 @@
+import Tracetest from '@tracetest/core';
```

Comment:
I think renaming `@tracetest/core` to `@tracetest/client` will make sense in the long run. We would only call the OSS product Tracetest Core. WDYT?